homeassistant/components/linkplay/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""Support for LinkPlay devices."""
+
+from dataclasses import dataclass
+import logging
+
+from linkplay.bridge import LinkPlayBridge
+from linkplay.discovery import linkplay_factory_bridge
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN, PLATFORMS
+
+__all__ = [
+    "DOMAIN",
+    "async_setup_entry",
+    "async_unload_entry",
+]
+
+
+@dataclass
+class LinkPlayData:
+    """Data for LinkPlay."""
+
+    bridge: LinkPlayBridge
```

Comment:
If you're only storing the bridge in there, you don't have to wrap it in a dataclass